flask/testing.py

```diff
@@ -20,7 +21,7 @@
     from urlparse import urlsplit as url_parse
 
 
-def make_test_environ_builder(app, path='/', base_url=None, *args, **kwargs):
+def make_test_environ_builder(app, path='/', base_url=None, json=None, *args, **kwargs):
```

Comment:
I realise this comment is a bit late but wouldn't it be better to not make json a keyword argument and instead check for it inside kwargs? This would mean that passing `json=None` would send a valid json request with data as 'null' instead of not doing anything. It also has the added benefit that you don't need to rename your import of `json`.